### ISSUE TITLE
Cannot set charset-utf8 ContentType on Invoke-Restmethod POST

### ISSUE THREAD

**Issue by gheorghiuradu** [PowerShell/PowerShell#3226]
Steps to reproduce
------------------
Run command Invoke-RestMethod [URI] -Mehtod POST -ContentType 'application/json; charset=utf-8' -Body $body

Expected behavior
-----------------
The command executes successfully 

Actual behavior
---------------
Command fails with error:
Invoke-RestMethod : The format of value 'application/json; charset=utf-8' is 
invalid.

If the contentype is set to 'application/json' only it works, but setting charset should be allowed, as it used to work on previous versions no problem.
Environment data
----------------
Ubuntu 16.04

```powershell
> $PSVersionTable
Name                           Value                                           
----                           -----                                           
PSVersion                      6.0.0-alpha                                     
PSEdition                      Core                                            
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                         
BuildVersion                   3.0.0.0                                         
GitCommitId                    v6.0.0-alpha.15                                 
CLRVersion                                                                     
WSManStackVersion              3.0                                             
PSRemotingProtocolVersion      2.3                                             
SerializationVersion           1.1.0.1   
```

**Comment by iSazonov** [comment]
It is external and It is CoreFX Issue https://github.com/dotnet/corefx/issues/16290

**Comment by ffeldhaus** [comment]
This was solved with #6018 and the introduction of `-SkipHeaderValidation`. This issue could be closed in my opinion.
